databags/sites.json

```diff
@@ -505,6 +505,16 @@
       "url": "https://archive.materialscloud.org",
       "screenshot_filename": "materials_cloud_archive.png",
       "types": ["research-data", "institutional-repository"]
+    },
+    {
+      "name": "Prism",
+      "description": "Prism is an institutional repository for the research and scholarly output of Northwestern Feinberg School of Medicine (FSM). Prism preserves intellectual works created by the Feinberg scholarly community, enhances the visibility of Feinberg scholarship, and promotes authors by enabling discovery and accessibility of works by the international scientific community.",
```

Comment:
Here is the modified text. We're just removing references to Feinberg and FSM:

'Prism is an institutional repository for the research and scholarly output at Northwestern University. Prism preserves intellectual works created by the scholarly community, enhances the visibility of scholarship, and promotes authors by enabling discovery and accessibility of works by the international scientific community.'

Reply:
Made edits here: https://github.com/inveniosoftware/inveniosoftware.org/pull/187 . You can double-check and I can actually merge directly.